plugins/core/webview/src/q-ui/components/login.vue

```diff
@@ -16,13 +15,10 @@
 
         <template v-if="stage === 'AUTHENTICATING'">
             <div class="font-amazon">
-                <div v-if="app === 'TOOLKIT' && profileName.length > 0" class="title">Authenticating...</div>
-                <div v-else>
-                    <div class="title bottom-small-gap">Authenticating in browser...</div>
-                    <div v-if="authorizationCode?.length !== 0" class="confirmation-code-container bottom-small-gap">
-                        <div class="hint">CONFIRMATION CODE</div>
-                        <div class="confirmation-code">{{ this.authorizationCode }}</div>
-                    </div>
+                <div class="title bottom-small-gap">{{ this.authenticatingText }}</div>
+                <div v-if="requireConfirmationCodeOrNot" class="confirmation-code-container bottom-small-gap">
+                    <div class="hint">CONFIRMATION CODE</div>
+                    <div class="confirmation-code">{{ this.authorizationCode }}</div>
                 </div>
                 <button
                     class="login-flow-button cancel-button font-amazon"
```

Comment:
For this button you can for now remove the `:disabled="!isAuthorizationInProgress`, so people can always exit the flow and start over when auth code is not there